src/frontend/screens/ObservationsList/NoProjectWarning.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import {StyleSheet, View} from 'react-native';
+import Warning from '../../images/Warning.svg';
+import {Text} from '../../sharedComponents/Text';
+import {defineMessages, useIntl} from 'react-intl';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import {useNavigationFromRoot} from '../../hooks/useNavigationWithTypes';
+import {BLACK, DARK_BLUE, LIGHT_GREY} from '../../lib/styles';
+import {ViewStyleProp} from '../../sharedTypes';
+
+const m = defineMessages({
+  cantShare: {
+    id: 'screens.ObservationsList.NoProjectWarning',
+    defaultMessage: 'You won’t be able to share observations collected yet.',
+  },
+  createOrJoin: {
+    id: 'screens.ObservationsList.createOrJoin',
+    defaultMessage: 'Create or Join a Project.',
+    description:
+      "The full sentence is 'To sync and share with other devices, Create or Join a Project.' The sentence needs to be seperated for styling purposes",
+  },
+  toSync: {
+    id: 'screens.ObservationsList.toSync',
+    defaultMessage: 'To sync and share with other devices,',
+    description:
+      "The full sentence is 'To sync and share with other devices, Create or Join a Project.' The sentence needs to be seperated for styling purposes",
+  },
+});
+
+export const NoProjectWarning = ({style}: {style?: ViewStyleProp}) => {
+  const {formatMessage} = useIntl();
+  const {navigate} = useNavigationFromRoot();
+  return (
+    <View style={[styles.container, style]}>
+      <Warning style={{marginRight: 10}} />
+      <View style={{flex: 1}}>
+        <Text style={{flexShrink: 1}}>{formatMessage(m.cantShare)}</Text>
+        <TouchableOpacity
+          style={{flexDirection: 'row', marginTop: 10}}
+          onPress={() => navigate('CreateOrJoinProject')}>
+          <Text style={{flexShrink: 1}}>
+            {formatMessage(m.toSync) + ' '}
+            <Text
+              style={{
+                textDecorationLine: 'underline',
+                color: DARK_BLUE,
+              }}>
+              {formatMessage(m.createOrJoin)}
+            </Text>
```

Comment:
We probably want to change the order of this for RTL languages like Arabic. Is that possible?

Reply:
Right. So this is the only place in the app that has one sentence with 2 parts of it differently styled. RN doesnt handle that well. You have to have 2 separate text boxes to make it work. 

I think this actually brings up a bigger issue that this might not work in terms of translations. We cannot guarantee the gramatical orderbof these 2 lines, or like you point out, RTL will be a problem. 

Ill flag this as an issue with design, but i think its ok to ignore for now.